urnx/types.go

```diff
@@ -0,0 +1,15 @@
+package urnx
+
+import "github.com/google/uuid"
+
+const PREFIX = "urn"
+const URNLENGTH = 4
```

Comment:
In general, unless we have a good reason to export constants I would rather them be unexported; capitalizing them like this feels unidiomatic and also has the side effect of exporting them for consumers.

Reply:
✅